kong/conf_loader.lua

```diff
@@ -321,6 +321,14 @@ local CONF_INFERENCES = {
   pg_max_concurrent_queries = { typ = "number" },
   pg_semaphore_timeout = { typ = "number" },
 
+  pg_ro_port = { typ = "number" },
+  pg_ro_timeout = { typ = "number" },
+  pg_ro_password = { typ = "string" },
+  pg_ro_ssl = { typ = "boolean" },
+  pg_ro_ssl_verify = { typ = "boolean" },
+  pg_ro_max_concurrent_queries = { typ = "number" },
+  pg_ro_semaphore_timeout = { typ = "number" },
```

Comment:
These need to be documented in `kong.conf.default`

Reply:
Added. Thanks!